Update hub-namespaces.adoc (#253)
uyuni-project/uyuni-docs: modules/large-deployments/pages/hub-namespaces.adoc

[[lsd-hub-namespaces]]
= Hub Namespaces

The Hub XMLRPC API operates in a similar way to the {productname} API.
For {productname} API documentation, see https://documentation.suse.com/suma.

The Hub XMLRPC API exposes the same methods that are available from the server's XMLRPC API, with a few differences in parameter and return types.
Additionally, the Hub XMLRPC API supports some Hub-specific end points which are not available in the {productname} API.

Hub supports three different namespaces:

* The ``hub`` namespace is used to target the Hub XMLRPC API Server.
It supports Hub-specific XMLRPC endpoints which are primarily related to authentication.
* The ``unicast`` namespace is used to target a single server registered in the hub.
It redirects any call transparently to one specific server and returns any value as if the server's XMLRPC API endpoint was used directly.
* The ``multicast`` namespace is used to target multiple servers registered in the hub.
It redirects any call transparently to all the specified servers and returns the results in the form of a ``map``.

> * By using no namespace any call is transparently redirected to the underlying {productname} Server XMLRPC API of the Hub Server.

Reviewer: ```suggestion
* If you do not specify a namespace, all calls are transparently redirected to the underlying {productname} Server XMLRPC API of the Hub Server.
```

Author: awesome, thanks

> Thus it is possible to call all the methods that are available on a {productname} Server XMLRPC API.

Reviewer: ```suggestion
This allows you to call all available methods on a {productname} Server XMLRPC API.
```
